lisa/runners/legacy_runner.py

```diff
@@ -363,7 +369,7 @@ class LogParser(InitializableMixin):
     # SetupConfig: { ARMImageName: Canonical 0001-com-ubuntu-server-focal 20_04-lts
     #  20.04.202102010, TestLocation: westus2 }
     CASE_IMAGE_LOCATION = re.compile(
-        r"SetupConfig: { ARMImageName: (?P<image>.+), TestLocation: (?P<location>.+) }"
+        r"SetupConfig: { ARMImageName: (?P<image>.+),.*TestLocation: (?P<location>.+) }"
```

Comment:
capture overwrite vmsize, here. The next CASE_VMSIZE pattern may not be matched.